applications/nrf5340_audio/prj_fota.conf

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# nRF5340 Audio
+CONFIG_NRF5340_AUDIO=y
+
+CONFIG_SAMPLE_RATE_CONVERTER=y
+CONFIG_SAMPLE_RATE_CONVERTER_FILTER_SIMPLE=y
+
+# General
+CONFIG_DEBUG=y
+CONFIG_DEBUG_INFO=y
+CONFIG_ASSERT=y
+CONFIG_STACK_USAGE=y
+CONFIG_THREAD_RUNTIME_STATS=y
+CONFIG_STACK_SENTINEL=y
+CONFIG_INIT_STACKS=y
+
+# Uart driver
+CONFIG_SERIAL=y
+
+# Logging
+CONFIG_LOG=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_LOG_TAG_MAX_LEN=2
+CONFIG_LOG_TAG_DEFAULT="--"
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_LOG_BUFFER_SIZE=4096
+
+# Use this for debugging thread usage
+#CONFIG_LOG_THREAD_ID_PREFIX=y
+
+# Console related defines
+CONFIG_CONSOLE=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=y
```

Comment:
Can you highlight the changes to this file?

Reply:
They sort of already are - `prj_fota.conf` is identical to `prj.conf`, except for the parts at the end that has the title "DFU" and "External Flash". Do you think it needs to be highlighted even more?